src/pages/components/messages/conversation.tsx

```diff
@@ -1,72 +1,81 @@
-import React,{useEffect, useState} from 'react'
+import React, {useEffect, useState} from 'react'
 import {Card} from 'primereact/card'
-import { ScrollPanel } from 'primereact/scrollpanel';
-import { InputText } from 'primereact/inputtext';
-import { Button } from 'primereact/button';
+import {ScrollPanel} from 'primereact/scrollpanel';
+import {InputText} from 'primereact/inputtext';
+import {Button} from 'primereact/button';
 import CreateMessage from './create-message';
-import { conversationProps,messageData } from '../../../types/conversationState';
+import {conversationProps, messageData} from '../../../types/conversationState';
 import './conversation.css'
 import HighlightMessage from '../highlight-message';
-export function Conversation(props:conversationProps){
-    const {messages,sendMessage,loadMessages}=props
-    
-    const [searchText,searchMessages]=useState('')
-    const stylePanel={ width: '100%', height: '400px' }
-  
- 
-    
-useEffect( ()=>{
-    loadMessages()
-     
-      },[loadMessages]
 
-)
+export function Conversation(props: conversationProps) {
+    const {messages, sendMessage, loadMessages} = props
 
-const messageComponentList=messages.map((msg:messageData)=>{
-    return (
-        <div  className='p-col p-col-fixed message-width' key={msg.id} >
-        <Card className={msg.isReceived?'received-message':'sent-message'}>
-            
-            <HighlightMessage message={msg.message} searchText={searchText}/>
-           
-        </Card>
-    </div>
+    const [searchText, searchMessages] = useState('')
+    const stylePanel = {width: '100%', height: '400px'}
+
+    useEffect(() => {
+            loadMessages()
+
+        }, [loadMessages]
     )
-}) 
-  
-    return(
-        <div className="p-grid">
-        <div className="p-col-3"></div>
-        <div className="p-col-9">
-        <Card className="conversation-title">
 
-            <div className="p-grid p-fluid">
-            <div className="p-col-12 p-md-4">
-            Manam
+    const messageComponentList = messages.map((msg: messageData) => {
+        return (
+            <div>
+                {
+                    !msg.isReceived && <div className="p-grid p-justify-end">
+                        <div className='p-col p-col-fixed' style={{width: '40%'}} key={msg.id}>
+                            <Card className={msg.isReceived ? 'received-message' : 'sent-message'}>
```

Comment:
redundant condition check msg.isReceived

Reply:
inline style for width can be moved to css file

Reply:
addressed the comments